src/data/migrations/20180104141345-remove-pool-poolMember-vote-models.js

```diff
@@ -0,0 +1,9 @@
+export async function up(r, conn) {
+  await r.tableDrop('pools').run(conn)
+  await r.tableDrop('poolMembers').run(conn)
+  await r.tableDrop('votes').run(conn)
```

Comment:
Could be slightly more performant with:

```js
Promise.all([
  r.tableDrop('pools').run(conn),
  r.tableDrop('poolMembers').run(conn),
  r.tableDrop('votes').run(conn),
])
```

...runs all of these async functions at the same time (in parallel) rather than one at a time (serially).